src/app/demo.component.css

```diff
@@ -0,0 +1,6 @@
+.app-demo {
```

Comment:
I would update the component and wrap everything in the template in `<div class="app-demo"></div>`